.github/workflows/coordinator-build-and-publish.yml

```diff
@@ -38,10 +40,14 @@ jobs:
     env:
       COMMIT_TAG: ${{ inputs.commit_tag }}
       DEVELOP_TAG: ${{ inputs.develop_tag }}
```

Comment:
Can we remove `DEVELOP_TAG` as well, it not uses anywhere, right?

Reply:
Instead of deleting the env var, I replaced the usage of inputs with the appropriate env var.